packages/workspace/src/core/project-graph/project-graph.spec.ts

```diff
@@ -131,6 +130,7 @@ describe('project graph', () => {
       ext: extname(f),
       mtime: 1
     }));
+    vol.reset();
```

Comment:
What does this do?

Reply:
it deletes all the files on the in-memory fs volume: https://github.com/streamich/memfs/blob/df7c6178811862a07f390c113188f615f57fa60c/README.md#volreset

I found out it was re-using the same dependencies cache file (and all the other files) between tests, so they could've influenced each other